.babelrc.js

```diff
@@ -0,0 +1,13 @@
+const { NODE_ENV, BABEL_ENV } = process.env;
+
+const cjs = BABEL_ENV === 'cjs' || NODE_ENV === 'test';
+
+module.exports = {
+  presets: [['env', { modules: false, loose: true }]],
+  plugins: [
+    cjs && 'transform-es2015-modules-commonjs',
```

Comment:
i dont use `loose` for cjs, because it "breaks" importing namespaces - it uses a simple assignment for `__esModule` property thus making it enumerable, and spreading an imported namespace onto some object carries over `__esModule` - gain of using loose in this case is minimal and I tend to avoid it for this case

Reply:
I actually just added the bottom two plugins, but missed the commonjs. I'll get this all merged on my end